### DISCUSSION TITLE
anyone decapped the one that supposedly has a supercap in it?

### DISCUSSION THREAD

**Discussion by ohsix** [cpetrich/counterfeit_DS18B20#31] [General]
as titled, seems wild, wanna see it

**Comment by cpetrich** [comment]
See photo on the web page for what you get by decapping. One would then need to etch off a layer to see the circuit.

**Comment by ohsix** [comment]
oh, i see, i thought it wouldn't be in that image since a supercap needs a high surface area and an organic solvent and stuff, did you mean a high value capacitor? (should be able to charge it rc circuit style to figure out the capacity)

thank you

**Comment by cpetrich** [comment]
well, ok, after reviewing Wikipedia I would concede that my wording is presumptive, thanks for pointing that out.
If you know about this sort of stuff, maybe you can somebody at Xi'an Supermicro Electronics Co., Ltd. to elaborate? Would be interesting.
